marketplaceItems/drawApp/fingerDraw_ui.html

```diff
@@ -0,0 +1,21 @@
+<html>
+    <head>
+        <title>Draw</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700" rel="stylesheet">
+        <link rel="stylesheet" href="fingerDraw_ui.css">
+    </head>
+    <body>
+        <h1>Thank you for downloading the draw app!</h1>
+        <p>This basic app allows you to draw lines with the index finger of your dominant hand in HMD using the trigger. 
+            In desktop, simply use your left mouse button to draw 1 meter in front of your camera view. The lines will 
+            disappear after 60 seconds unless erased in HMD mode using the grip. You will need to close your tablet to 
+            draw. The app is intended to help users who cannot use their microphone or as a quick visual aid during 
+            conversation.</p>
+            <p>If you would like to remove this app from your tablet and toolbar, open your running scripts (CTRL + j) 
+                and close 'fingerDraw_app.js'. If you ever want to reinstall, it can be found for free on the marketplace.</p>
+            <p>You will not see this page again unless you redownload the app from the marketplace or your inventory. 
```

Comment:
the user could see this page again if they used the "Open" button from within Inventory if they downloaded the item from Marketplace!